src/App.tsx

```diff
@@ -62,7 +62,10 @@ function App() {
         path="/messages/:userId1/:userId2"
         element={<ProtectedRoute component={DirectMessage} />}
       />
-      <Route path="/profile" element={<ProtectedRoute component={Profile} />} />
+      <Route
+        path="/user/:username"
```

Comment:
Do we need `/user/:username`? Or can we simplify it to `/:username`?

Reply:
Yeah good point, I'll simplify this route